static/app/components/events/eventEntries.tsx

```diff
@@ -6,6 +6,7 @@ import uniq from 'lodash/uniq';
 
 import {addErrorMessage} from 'sentry/actionCreators/indicator';
 import ErrorBoundary from 'sentry/components/errorBoundary';
+import {Evidence} from 'sentry/components/events/evidence';
```

Comment:
Should this be called `EventEvidence` and be down here?
![image](https://user-images.githubusercontent.com/1421724/214176233-02460e5b-da6f-476a-903a-cdc60523afe7.png)

Reply:
Good thinking, renamed